content/whats-next/whats-next-short-term.json

```diff
@@ -1,16 +1,10 @@
 [
   {
-    "title": "Shared Catalogs - PDF exports for your product selection",
-    "description": "Shared Catalog users will be able to download their product selection in a PDF format. This will bring even more autonomy to users while allowing them to share clear and structured data to their stakeholders. Sharing a product catalog will become a lot faster as the time spent reworking exported files will be reduced drastically.",
-    "img": "./img/PDFExport.png",
-    "areas": ["Collaboration"]
-  },
-  {
-    "title": "Akeneo PIM - Tailored Exports for Growth Edition",
-    "description": "The ability to export customized flat files from Akeneo PIM will be possible in Akeneo Growth Edition. As a GE user, sharing key product data with your distributors will become radically easier with this powerful and entirely customizable capability. Define the data, customize the structure, perform operations and more, all for a seamless exportation experience.",
-    "img": "./img/TailoredExport-WhatsNext.png",
+    "title": "Akeneo PIM - Tailored Imports and Data Mapping",
+    "description": "Being able to import and map data coming from manufacturers or suppliers and leverage it efficiently in Akeneo PIM is essential. It will be possible to import various XLSX file structures, to map columns to their PIM attributes, and to define operations to transform the data if needed (data cleansing, concatenation, split,...).", with Tailored Imports.
```

Comment:
```suggestion
    "description": "Being able to import and map data coming from manufacturers or suppliers and leverage it efficiently in Akeneo PIM is essential. It will be possible to import various XLSX file structures, to map columns to their PIM attributes, and to define operations to transform the data if needed (data cleansing, concatenation, split,...) with Tailored Imports".
```